app/releasemanager.cpp

```diff
@@ -20,7 +20,9 @@
 #include "releasemanager.h"
 #include "drivemanager.h"
 
-#include "isomd5/libcheckisomd5.h"
+extern "C" {
+#include <libcheckisomd5.h>
```

Comment:
A lot (I'd say most, even) of C libraries do `#ifdef __cplusplus` with this at the start and end of their public headers. I don't see a big problem in this though.